.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+npm-debug.log
```

Comment:
awesome, ta. I figured the codebase has gotten into a bit of a state so
something had to be done sooner or later and if it was left it would only
deteriorate more. Maybe it's worth keeping it until the next major release
though...

On Sun, Apr 19, 2015 at 12:33 AM, Ben Evans <ben@bensbit.co.uk> wrote:

> good shout, I've pushed a fix
>
>
> On Sat, Apr 18, 2015 at 6:15 PM, Marshall T. Rose <
> notifications@github.com> wrote:
>
> In .gitignore
> <https://github.com/bencevans/node-sonos/pull/81#discussion_r28645744>:
>
> > @@ -1 +1,2 @@
> >  node_modules
> > +npm-debug.log
>
> personally, i always prefer *.log
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bencevans/node-sonos/pull/81/files#r28645744>.
>
>